setup.cfg

```diff
@@ -0,0 +1,30 @@
+[metadata]
+name = pyflow
+version = attr: pyflow/version.__version__
+author = European Centre for Medium-Range Weather Forecasts (ECMWF)
+author_email = software.support@ecmwf.int
+license = Apache 2.0
+license_files = LICENSE
+description = Create pythonic ecFlow suites
+long_description = file: README.md
+long_description_content_type=text/markdown
+url = https://pyflow-workflow-generator.readthedocs.io/en/
+
+[options]
+packages = find_namespace:
```

Comment:
Any particular reason why find_namespace instead of find? Not sure it matters much though.

Reply:
Not really, sorry. I used another project that uses Python namespaces (PEP 382). Should work with `find` as I believe pyflow doesn't use namespaces.

Reply:
Done.